aws/ecs/service-no-alb-with-volume/README.md

```diff
@@ -0,0 +1,50 @@
+# aws/ecs/service-no-alb - EC2 Container Service Service/Task without load balancer
+This module is used to create an ECS service as well as task definition
+
+## What this does
+
+ - Create task definition
+ - Create service
+
+## Required Inputs
+
+ - `cluster_id` - ID for ECS Cluster
+ - `service_name` - Name of service, all lowercase, no spaces.
+ - `service_env` - Name of environment, used in naming task definition. Ex: `staging`
+ - `container_def_json` - JSON for container definition.
+ - `desired_count` - Number of tasks to run in service
+ - `volume_name` - Name for volume
+ - `volume_host_path` - Path on host EC2 instance to mount volume
+
+### Optional Inputs
```

Comment:
Your vars.tf file includes a lot of values that aren't included here.

Reply:
removed unused vars